vernac/comInductive.mli

```diff
@@ -42,6 +42,31 @@ val do_mutual_inductive
 
 val make_cases : Names.inductive -> string list list
 
+(** inductive_expr at the constr level *)
+type mutual_inductive_declaration = {
+  mie : Entries.mutual_inductive_entry;
+  univ_binders : UnivNames.universe_binders;
+  implicits : DeclareInd.one_inductive_impls list;
+  uctx : Univ.ContextSet.t;
+  where_notations : Metasyntax.where_decl_notation list;
+  coercions : Libnames.qualid_r CAst.t list;
+}
```

Comment:
Please put this into a record so fields are qualified. `MInd_declaration.t` maybe?